tests/e2e/scenarios/ha/test.yml

```diff
@@ -0,0 +1,237 @@
+          ansible.builtin.include_role:
+            name: e2e.tests.generate_namespaces
+
+        - name: "[ HATest - Setup ] Creating Skupper resources on east_ha namespace"
+          skupper.v2.resource:
+            kubeconfig: "{{ kubeconfig }}"
+            path: "{{ item }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+          with_items:
+            - "{{ playbook_dir }}/resources/east_ha/backend.yaml"
+            - "{{ playbook_dir }}/resources/east_ha/site.yaml"
+            - "{{ playbook_dir }}/resources/east_ha/connector.yaml"
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Creating Skupper resources on west_ha namespace"
+          skupper.v2.resource:
+            kubeconfig: "{{ kubeconfig }}"
+            path: "{{ item }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+          with_items:
+            - "{{ playbook_dir }}/resources/west_ha/site.yaml"
+            - "{{ playbook_dir }}/resources/west_ha/listener.yaml"
+          when:
+            - "'west' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Issue a Skupper access token from west_ha namespace"
+          skupper.v2.token:
+            kubeconfig: "{{ kubeconfig }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+            name: west-ha-grant
+            redemptions_allowed: 1
+          register: west
+          when:
+            - "'west' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Apply token to east site"
+          skupper.v2.resource:
+            kubeconfig: "{{ kubeconfig }}"
+            def: "{{ hostvars['west_ha']['west']['token'] }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Check the link status"
+          shell: kubectl get links -n  {{ namespace_prefix }}-{{ namespace_name }} --kubeconfig {{ kubeconfig }}
+          register: links_status
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Display the output of wait"
+          ansible.builtin.debug:
+            var: links_status
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Wait for links to get in Ready state"
+          shell: kubectl wait --kubeconfig {{ kubeconfig }} --for=condition=Ready=true links --all -n {{ namespace_prefix }}-{{ namespace_name }} --timeout=30s
+          register: links_status
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: "[ HATest - RunTest ] Create the job yaml, based on template, setting the runtime to {{ locust_runtime }}"
+          template:
+            src: "{{ playbook_dir }}/templates/locust-job.yaml.j2"
+            dest: "{{ playbook_dir }}/resources/west_ha/locust-job.yaml"
```

Comment:
Should the template be rendered here or in a temp directory to the local repo from getting dirty?